core/src/main/java/org/apache/calcite/adapter/enumerable/RexImpTable.java

```diff
@@ -425,6 +425,7 @@
 import static org.apache.calcite.sql.fun.SqlStdOperatorTable.IS_JSON_SCALAR;
 import static org.apache.calcite.sql.fun.SqlStdOperatorTable.IS_JSON_VALUE;
 import static org.apache.calcite.sql.fun.SqlStdOperatorTable.IS_NOT_A_SET;
+import static org.apache.calcite.sql.fun.SqlStdOperatorTable.IS_NOT_DISTINCT_FROM;
```

Comment:
what about IS_DISTINCT_FROM?
Does that need an implementor, or is it converted to IS_NOT_DISTINCT_FROM?

Reply:
IS_DISTINCT_FROM will expand to `x=y IS NOT TRUE and ((x is not null) or (y is not null))`. There is currently no method to collapse it, so it may not be necessary to implement it temporarily?